### PR TITLE
Update renfe_ids and add Spanish stations

### PR DESCRIPTION
**Pull request by jamesparkerttl**
Data source: renfe id's are obtained from data publicly available on the renfe website

Added UIC code for Elx AV 7103410

Updated renfe_id on the following ES stations:
| Station | renfe_id |
|--------|--------|
| Loja | 05012 |
| Cambrils | 65422 |
| L’Hospitalet de l’Infant | 65420 |
| Mirasierra Paco De Lucía | 97200 |
| Roca-Cúper | 65001 |
| Moncofa | 65205 |
| Caudiel | 67211 |
| Sant Vicent Centre | 60913 |
| Universidad de Alicante | 60914 |
| Aljaima | 54407 |
| Málaga Centro Alameda | 54517 |
| Hernani Centro | 11507 |
| Herrera | 11514 |

Added these ES Stations
| Station | renfe_id |
| Antequera AV | 02030 |
| A Gudina-Porta Galicia | 08245 |
| allosa de Segura Cox | 03412 |

### CODE REVIEW

**Review by friberah — APPROVED** [review]

File: stations.csv [hunk 1]
```diff
@@ -70497,3 +70497,6 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 74638;Bari Santa Rita;bari-santa-rita;;;41.07330246992781;16.851540354841326;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8311250;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74638;
 74639;Lascari;lascari;;;38.011695;13.928291;19906;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;S12070;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74639;
 74640;Torino Grosseto;torino-grosseto;;;45.10546665758956;7.666751284086174;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8300097;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74640;
+74641;Antequera Av;antequera-av;7102030;;37.02978;-4.56119;;ES;Europe/Madrid;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;02030;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74641;
+74642;A Gudina-Porta Galicia;a-gudina-porta-galicia;7108251;;42.06367;-7.13436;;ES;Europe/Madrid;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;08251;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74642;
```
Comment (ConscritNeuneu) [inline stations.csv:70501]: image

```suggestion
74642;A Gudiña – Porta de Galicia;a-gudina-porta-de-galicia;7108251;;42.06367;-7.13436;;ES;Europe/Madrid;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;08251;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74642;
```

File: stations.csv [hunk 2]
```diff
@@ -70497,3 +70497,6 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
...
+74643;Callosa de Segura Cox;callosa-de-segura-cox;7103412;;38.136389;-0.875833;;ES;Europe/Madrid;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;03412;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74643;
```
Comment (ConscritNeuneu) [inline stations.csv:70502]: Even if Wikipedia suggests a `-` between Callosa de Segura and Cox I think it's better typographically to use an EN-dash.

```suggestion
74643;Callosa de Segura – Cox;callosa-de-segura-cox;7103412;;38.136389;-0.875833;;ES;Europe/Madrid;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;03412;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74643;
```

File: stations.csv [hunk 3]
```diff
@@ -70497,3 +70497,6 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 74638;Bari Santa Rita;bari-santa-rita;;;41.07330246992781;16.851540354841326;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8311250;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74638;
 74639;Lascari;lascari;;;38.011695;13.928291;19906;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;S12070;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74639;
 74640;Torino Grosseto;torino-grosseto;;;45.10546665758956;7.666751284086174;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8300097;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74640;
+74641;Antequera Av;antequera-av;7102030;;37.02978;-4.56119;;ES;Europe/Madrid;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;02030;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74641;
```
Comment (ConscritNeuneu) [inline stations.csv:70500]:
```suggestion
74641;Antequera AV;antequera-av;7102030;;37.02978;-4.56119;;ES;Europe/Madrid;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;02030;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74641;
```

File: stations.csv (hunk 1 above)
Comment (friberah) [reply stations.csv:70501]: LOL I've missed this one...

Comment (ConscritNeuneu) [reply stations.csv:70501]: I was about to comment that this should be reviewed by somebody more local to Spain. Stations is a detective's game, locals have a head start.

**Review by ConscritNeuneu — APPROVED** [review]

**Comment by ConscritNeuneu** [comment]
Could you clean-up your history please?

**Comment by jamesparkerttl** [comment]
> Could you clean-up your history please?

Done @ConscritNeuneu

**Pull request merged**